### PR TITLE
Add heroku compatibility

### PR DESCRIPTION
**Pull request by marvinschopf**
@jens1o 

I've added heroku compatibility by using the port set in an environment variable or port `8080` if no specific port is set.

Please review this PR as soon as possible.

Thank you,
MagicMarvMan

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -13,6 +13,9 @@ const TrashRouteHandler = require('./routes/trash');
 const TrashcanRouteHandler = require('./routes/trashcan');
 const ScoreRouteHandler = require('./routes/score');
 
+// Use environment port specification if set; if not, use port 8080 (makes app compatible with heroku)
+const PORT = process.env.PORT || 8080
```
Comment (jens1o) [inline src/main.js]: use `Number.parseInt` for the env var, it is a string otherwise.

**Review by jens1o — APPROVED** [review]

**Review by jens1o — CHANGES_REQUESTED** [review]
wrong button

File: src/main.js (hunk 1 above)
Comment (marvinschopf) [reply src/main.js]: fixed!

**Comment by marvinschopf** [comment]
fixed ^^

**Review by jens1o — APPROVED** [review]

**Pull request merged**